app/src/main/res/layout/faq_single_activity.xml

```diff
@@ -20,7 +20,7 @@
         android:id="@+id/faq_single_activity_toolbar"
         android:layout_width="match_parent"
         android:layout_height="?attr/actionBarSize"
-        android:background="?attr/colorPrimary"
+        android:background="@color/color_def_oppia_green"
```

Comment:
ditto.

Reply:
> @bhaktideshmukh Thanks, overall this looks good. Just had a few suggestions. Also, I think the content of the FAQ and other list items in FAQ questions list should be using `sub_text color` (which is defined in color_defs as `color_def_oppia_silver`.) and not the Body Header color, you might need to confirm this from the mocks and let me know if It is actually mentioned otherwise in the mocks.

Content of FAQ in mocks is mentioned as 'oppia_primary_text'.